api/v1alpha1/securitypolicy_types.go

```diff
@@ -91,7 +92,13 @@ type SecurityPolicySpec struct {
 	// GraphQL                       map[string]GraphAccessDefinition `json:"graphql_access_rights"`
 }
 
-// from tyk/session.go
+// GraphQLType represents a GraphQL Type for Tyk.
+//+k8s:deepcopy-gen:interfaces=github.com/TykTechnologies/graphql-go-tools/pkg/graphql.Type
```

Comment:
It will work without this annotation too

Reply:
Thanks! Removed in b2095bfd75716efbff6aefa8d29c898d66ade395